pkg/api/customization/cluster/actions_cis.go

```diff
@@ -62,6 +62,10 @@ func (a ActionHandler) runCisScan(actionName string, action *types.Action, apiCo
 		return httperror.WrapAPIError(err, httperror.ClusterUnavailable,
 			fmt.Sprintf("cluster not ready"))
 	}
+	if cluster.Spec.WindowsPreferedCluster {
+		return httperror.WrapAPIError(err, httperror.InvalidAction,
+			fmt.Sprintf("cannot run scan on a windows cluster"))
+	}
```

Comment:
@leodotcloud do you think we can move this check at top and avoid all other checks sice with Windows we can right away exit

Reply:
👍